### PR TITLE
Fixes failing kitchensink examples

### PR DESCRIPTION
**Pull request by reaktivo**
### Changes

- Updated the kitchensink example readme to include complete instructions to run it
- Updated the kitchensink package.json with a correct reference to webpack

### CODE REVIEW

**Comment by fahad19** [comment]
thanks @reaktivo!

**Comment by fahad19** [comment]
Looks like it's happening since #161 was merged. I am happy to merge this PR as is.

---

But for a proper one later, may be add kitchensink's own `package.json` in `examples/kitchensink/package.json`, and list all the required dependencies there.

File: examples/kitchensink/app-counter/package.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
   "description": "Kitchensink counter app in Frint",
   "main": "index.js",
   "scripts": {
-    "build": "webpack",
+    "build": "../../../node_modules/.bin/webpack",
```
Comment (jackTheRipper) [inline examples/kitchensink/app-counter/package.json]: if doing so then let's remove the reference to Webpack in this 'package.json' file

**Comment by jackTheRipper** [comment]
@reaktivo following has been offered by @fahad19 I offer the following: https://github.com/Travix-International/frint/pull/176

**Comment by reaktivo** [comment]
I'm closing this one since it has been resolved by #176

**Pull request closed without merging**